lib/repository_fetching_worker.rb

```diff
@@ -0,0 +1,10 @@
+class RepositoryFetchingWorker < BaseWorker
+  sidekiq_options retry: false, queue: 'default'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.